modules/react/text/lib/Text.tsx

```diff
@@ -1,76 +1,100 @@
 import * as React from 'react';
+import {base, system} from '@workday/canvas-tokens-web';
 import {createComponent} from '@workday/canvas-kit-react/common';
-// type changes no need
-import {type, CanvasTypeHierarchy, CanvasTypeVariants} from '@workday/canvas-kit-react/tokens';
-import {Box, BoxProps} from '@workday/canvas-kit-react/layout';
+import {createModifiers, createStyles, CSProps} from '@workday/canvas-kit-styling';
+import {mergeStyles, CommonStyleProps} from '@workday/canvas-kit-react/layout';
 
-export interface TextProps extends BoxProps {
+type TypeSize = 'large' | 'medium' | 'small';
+type TokenName = `${keyof typeof system.type}.${TypeSize}`;
+
+export interface TextProps extends CSProps, CommonStyleProps {
```

Comment:
` CSProps, CommonStyleProps` this type seems common enough for us to create a type to extend this

Reply:
I've replaced this type by `BoxProps` for now. It can be created as a separated type in the future